README.md

```diff
@@ -1 +1,11 @@
-# laura.valtere
\ No newline at end of file
+# About me
+![This is an image] (/laura.valtere/img/biker.png)
```

Comment:
Incorrect path. Should be something like (/img/biker.png). Please do the update, and check locally if it is working before committing the new changes.

Reply:
done :)